src/prefect/utilities/logging.py

```diff
@@ -109,7 +109,9 @@ def _write_logs(self) -> None:
             try:
                 while self.pending_length < max_batch_length:
                     log = self.queue.get_nowait()
-                    self.pending_length += sys.getsizeof(log)
+                    self.pending_length += sys.getsizeof(
+                        log, len(log.get("message", "")) + 150
```

Comment:
Can you explain this line a bit more? What does the 150 represent?

Reply:
I've actually learned that `getsizeof` doesn't include the size of string literal values and the old code is just wrong :'( the tests were passing because we didn't have any strings that were large enough to exceed the overhead. I've rewritten this.